package.json

```diff
@@ -97,6 +97,7 @@
     "hoist-non-react-statics": "^3.0.1",
     "instabug-reactnative": "^9.1.6",
     "io-ts": "1.8.5",
+    "io-react-native-zendesk": "https://github.com/pagopa/io-react-native-zendesk.git#0.3.0",
```

Comment:
Since we merged 2 PR
https://github.com/pagopa/io-react-native-zendesk/pulls?q=is%3Apr+is%3Aclosed
we could create a new tag (0.3.1) to get the last changes.
what do you think?

Reply:
Yes, make sense!
Updated in [8f610c0](https://github.com/pagopa/io-app/pull/3462/commits/8f610c0edc674be30c9120ff79b1918dc76ba5ac)